pkgs/by-name/gh/gh-copilot/package.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, lib
+, fetchurl
+, system
+}:
+let
+  systemToBinary = {
+    "x86_64-linux" = {
+      name = "linux-amd64";
+      hash = "12mysfa84fzq3f95mi6i46yrcbdl3y91sn53snr4qqhm2w8yzb0l";
+    };
+    "aarch64-linux" = {
+      name = "linux-arm64";
+      hash = "14yd5h7fncq2aamxvcfjb5wsi64gs16gs2zj6gihg9ifglxgvxg2";
+    };
+    "x86_64-darwin" = {
+      name = "darwin-amd64";
+      hash = "0888gis7f2frziiskrws9fzfjshzydr14j8kapfqx8qcgcljb0av";
+    };
+    "aarch64-darwin" = {
+      name = "darwin-arm64";
+      hash = "1whywjjdzppswij3hrq3avgyhl363c4l5a4x5lixk9kzm3mq0qqp";
+    };
+  };
+in
+stdenv.mkDerivation rec {
+  pname = "gh-copilot";
+  version = "0.5.4-beta";
+
+  src = fetchurl {
+    name = "gh-copilot";
+    url = "https://github.com/github/gh-copilot/releases/download/v${version}/${systemToBinary.${system}.name}";
+    sha256 = systemToBinary.${system}.hash;
```

Comment:
```suggestion
    hash = systemToBinary.${system}.hash;
```

Reply:
Done, and also replaced hashes with base64 variant